client/packages/system/src/Asset/utils.ts

```diff
@@ -108,9 +104,7 @@ export const importRowToCsv = (
       node.model,
       node.class,
       node.category,
-    ].concat(
-      Object.values(node?.properties ?? {}).map(property => property.value)
-    );
+    ].concat(props.map(key => node.properties?.[key] ?? ''));
```

Comment:
as noted. adding in `uniqBy` or similar to avoid dupes here 🙏

Reply:
[350825f](https://github.com/msupply-foundation/open-msupply/pull/4001/commits/350825f1b4757a1596b505251c0e36ba7a6329b2)